agent/stats/engine.go

```diff
@@ -102,6 +105,18 @@ func (resolver *DockerContainerMetadataResolver) ResolveTask(dockerID string) (*
 	return task, nil
 }
 
+func (resolver *DockerContainerMetadataResolver) ResolveTaskByARN(taskArn string) (*apitask.Task, error) {
+	if resolver.dockerTaskEngine == nil {
+		return nil, fmt.Errorf("Docker task engine uninitialized")
```

Comment:
(probably from copy paste but) error message start with lowercase, same for line 114